update load base split config (#5098)
pingcap/docs: dynamic-config.md

| `{db-name}.{cf-name}.titan.blob-run-mode` | The mode of processing blob files |
| `storage.block-cache.capacity` | The size of shared block cache (supported since v4.0.3) |
| `backup.num-threads` | The number of backup threads (supported since v4.0.3) |

> | `split.qps-threshold` | The threshold to execute `load-base-split` on a Region. If the QPS of a read request for a Region exceeds the QPS threshold for a consecutive period of time, this Region should be split.|

Reviewer: ```suggestion
| `split.qps-threshold` | The threshold to execute `load-base-split` on a Region. If the QPS of read requests for a Region exceeds `qps-threshold` for a consecutive period of time, this Region should be split.|
```

> | `split.byte-threshold` | The threshold to execute `load-base-split` on a Region. If the traffic of a read request for a Region exceeds the byte-threshold for a consecutive period of time, this Region should be split. |

Reviewer: ```suggestion
| `split.byte-threshold` | The threshold to execute `load-base-split` on a Region. If the traffic of read requests for a Region exceeds the `byte-threshold` for a consecutive period of time, this Region should be split. |
```

> | `split.split-balance-score` | The parameter of `load-base-split`, which ensures the load of the two split Regions is as balanced as possible. The smaller the value is, the more balanced the load is, but it might also cause the split failure. |

Reviewer: ```suggestion
| `split.split-balance-score` | The parameter of `load-base-split`, which ensures the load of the two split Regions is as balanced as possible. The smaller the value is, the more balanced the load is. But setting it too small might cause the split failure. |
```

> | `split.split-balance-score` | The parameter of `load-base-split`, which ensures the load of the two split Regions is as balanced as possible. The smaller the value is, the more balanced the load is. But setting it too small might cause the split failure. |

Reviewer: ```suggestion
| `split.split-balance-score` | The parameter of `load-base-split`, which ensures the load of the two split Regions is as balanced as possible. The smaller the value is, the more balanced the load is. But setting it too small might cause split failure. |
```
